internal/cmd/cloud-signup/command.go

```diff
@@ -40,174 +31,22 @@ func New(prerunner pcmd.PreRunner, userAgent string, ccloudClientFactory pauth.C
 		clientFactory: ccloudClientFactory,
 		isTest:        isTest,
 	}
-	cmd.RunE = c.cloudSignupRunE
-
-	cmd.Flags().String("url", "https://confluent.cloud", "Confluent Cloud service URL.")
+	cmd.RunE = c.signup
 
 	return cmd
 }
 
-func (c *command) cloudSignupRunE(cmd *cobra.Command, _ []string) error {
-	url, err := cmd.Flags().GetString("url")
+func (c *command) signup(cmd *cobra.Command, _ []string) error {
+	utils.Println(cmd, "You will now be redirected to the Confluent Cloud sign up page in your browser.")
+	err := form.ConfirmEnter(cmd)
 	if err != nil {
```

Comment:
```suggestion
	if err := form.ConfirmEnter(cmd); err != nil {
```

Reply:
I just did that so I wouldn't have to use `:=` again on line 46. I can't change both though.